docs/quick-start.md

```diff
@@ -298,8 +298,17 @@ values to match against, and can contain multiple variants.
 
 ## Local Declarations
 
-TODO
+A **local** declaration binds a variable to the value of an expression. `.local` is like `const` in JavaScript. Think of it as a convenient way to reuse the result of an operation without repeating it.
+
+```mf2
+.local $x = {|This is an expression|}
+.local $y = {$now :datetime dateStyle=long}
+```
```

Comment:
Can you make this an actual runnable example? Right now it is missing the quoted pattern body.